examples/demo-windows.yaml

```diff
@@ -0,0 +1,34 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: podpvc-windows
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: csi-gce-pd
```

Comment:
Do we need storageclass with ntfs fstype?

Reply:
before we don't need to. but after controller.yaml is changed to have default fsType to ext4, I think we need fstype in storageclass for Windows